Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -236,7 +238,7 @@ - (NSDictionary *)liveContext
     context[@"network"] = ({
         NSMutableDictionary *network = [[NSMutableDictionary alloc] init];
 
-        if (self.bluetooth.hasKnownState)
+        if (self.bluetooth && self.bluetooth.hasKnownState)
```

Comment:
FWIW in Objc if `self.bluetooth` is `nil` then `self.bluetooth.hasKnownState` will evaluate to `NO` automatically. It's kind of an idiomatic objc thing, but of course both way works.